README-QWIKLABS.md

```diff
@@ -191,8 +191,7 @@ Once the app has been deployed, it can be viewed in the [Workload](https://conso
 Incidentally, the endpoint can be programmatically acquired using the following command:
 
 ```console
-echo http://$(kubectl get svc tracing-demo -n default \
-  -ojsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo http://$(kubectl get svc tracing-demo -n default -ojsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
Recommend wrapping in quotes like: `echo "http://$(kubectl get svc tracing-demo -n default -ojsonpath='{.status.loadBalancer.ingress[0].ip}')"`

Reply:
Thanks for the feedback! I've committed the change you recommended.